app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java

```diff
@@ -55,4 +56,19 @@ public void onClickCancelLAOTest() {
         onView(withId(R.id.button_cancel_launch)).perform(click());
         onView(withId(R.id.fragment_home)).check(matches(isDisplayed()));
     }
+
+    //These tests only pass when testingAttendee is true in MainActivity
```

Comment:
You can create a function in MainActivity to set testingAttendee to true at the beginning of each test